app/Services/Manager.php

```diff
@@ -148,6 +148,9 @@ public function getUserActivity($id, $model)
             $signup->reportback->updated_at = new Carbon($signup->reportback->updated_at);
             $signup->reportback->updated_at = $signup->reportback->updated_at->format('Y-m-d');
 
+            // Return set flagged status to 'pending' if it is false.
+            $signup->reportback->flagged = ($signup->reportback->flagged) ? $signup->reportback->flagged : 'pending';
```

Comment:
Just curious, why did you have to wrap it in parentheses? Pre-emptive strike against possible issue or was there a specific issue?